### ISSUE TITLE
Backslashes are being escaped too many times

### ISSUE THREAD

**Issue by cmgrayb** [kellystuard/terraform-provider-tss#6]
### Terraform Version
0.14.9

### Affected Resource(s)
data.tss_secret_field

### Terraform Configuration Files
```hcl
data tss_secret_field fieldname {
  number = (int)
  slug   = "(fieldname)"
}
```

### Expected Behavior
Field data should be read as-is and escaped only enough to reproduce the originating string

### Actual Behavior
Backslashes are doubled as escape characters by both the API and the provider.  As such, a field in TSS with firststring\\laststring such as a domain user credential is returned by the API as firststring\\\\laststring and then escaped again to firststring\\\\\\\\laststring as seen in state, resulting in Terraform consuming the value elsewhere as firststring\\\\laststring

**Comment by kellystuard** [comment]
Replicated. Seems the response is coming back from the API as a JSON-encoded string. Need to decode the response before storing it.

**Comment by kellystuard** [comment]
Fixed in Release v10.9.37.
